engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -2047,6 +2047,8 @@ class SemanticsObject {
       // IMPORTANT: because headings also cover certain kinds of headers, the
       //            `heading` role has precedence over the `header` role.
       return EngineSemanticsRole.heading;
+    } else if (isLink) {
+      return EngineSemanticsRole.link;
```

Comment:
Can we put this immediately above `isButton`? I don't think it makes a difference, but it's easier to understand when they are next to each other.